chips/nrf52832/src/chip.rs

```diff
@@ -0,0 +1,12 @@
+use crate::interrupt_service::Nrf52832InterruptService;
+use kernel::static_init;
+use nrf52::chip::NRF52;
+
+pub unsafe fn new() -> &'static NRF52<Nrf52832InterruptService> {
+    let interrupt_service = static_init!(Nrf52832InterruptService, Nrf52832InterruptService::new());
+    let chip = static_init!(
```

Comment:
This would be the first/only use of `static_init` outside the boards crates, which is really where it belongs. Is it here just because there are two? Is there a way to avoid that by, e.g., have the chip own the `interrupt_service` rather than own a static reference?

Reply:
I moved the `static_init` here to address @bradjc's comment https://github.com/tock/tock/pull/1505#issuecomment-565069558 but indeed I didn't realize that `static_init` out of boards was a new pattern. I don't see any issue with refactoring that as long as it works.

Reply:
I figured out how to fix this.